assets/i18n/en_US.json

```diff
@@ -134,7 +134,7 @@
     },
     "patchItem": {
         "unsupportedDialogText": "Selecting this patch may result in patching errors.\n\nApp version: {packageVersion}\nSupported versions:\n{supportedVersions}",
-        "unsupportedPatchVersion": "Patch is not supported for this app version. Enable the experimental toggle in settings to proceed.",
+        "unsupportedPatchVersion": "Patch is not supported for this app version. Disable the compatibility check in settings to proceed.",
```

Comment:
I think there's no reason to instruct the user to bypass the checks, we don't want them doing that, so I believe that half of the string can be removed

Reply:
It also is given in a toast which can't fit both sentences in it on most devices. Removed